src/main/java/net/tropicraft/core/common/block/experimental/TropicraftExperimentalLeaveBlock.java

```diff
@@ -0,0 +1,117 @@
+package net.tropicraft.core.common.block.experimental;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.tags.BlockTags;
+import net.minecraft.world.level.LevelAccessor;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.LeavesBlock;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.StateDefinition;
+import net.minecraft.world.level.block.state.properties.BooleanProperty;
+import net.minecraft.world.level.block.state.properties.IntegerProperty;
+
+import java.util.Random;
+
+public class TropicraftExperimentalLeaveBlock extends LeavesBlock {
+
+    public static IntegerProperty EXTRADISTANCE;
+    public static BooleanProperty TEMPIGNORERANDOMTICK; //Property to check if there has even been a block update to prevent unwanted decay instantly
+
+    public TropicraftExperimentalLeaveBlock(Properties settings) {
+        super(settings);
+        this.registerDefaultState(this.defaultBlockState().setValue(DISTANCE, 7).setValue(EXTRADISTANCE, 20).setValue(TEMPIGNORERANDOMTICK, false));
```

Comment:
At the time, I was worrired that due to the class inheriting the Base Leaf Block, It could cause issue if somewhere internally the block was then set with DISTANCE property causing a crash or exception due to it missing such

Reply:
Will need to make sure that this doesn't cause any Major Incompatibility with other mods attempting to get DISTANCE data. Maybe I could somehow interpreted the call to get the DISTANCE from a TropicratExperimentalLeaveBlock (Better name is needed), as wanting the CUSTOM_DISTANCE value just interpolated to 1-7 scale instead of 1-20?